pkl-core/src/test/files/LanguageSnippetTests/input/api/string.pkl

```diff
@@ -18,6 +18,12 @@ facts {
     !str1.isBlank
   }
 
+  ["isBase64"] {
+    "".isBase64
+    "AQIDBA==".isBase64
+    !"hello there".isBase64
+  }
```

Comment:
```suggestion
  }
  ["toBytesWithCharset"] {
    "Hello".toBytesWithCharset("UTF-8").length == 5
    "Hello".toBytesWithCharset("UTF-16").length > 5
  }
```

Reply:
FYI: this is just a draft PR! This will be fleshed out after the SPICE has been generally agreed to.